### ISSUE TITLE
Question about the dataloader on kinetics 400

### ISSUE THREAD

**Issue by andrewwang0612** [apple/ml-cvnets#35]
Hi，thanks for such a great job!
I download the kinetics 400  from this github
**https://github.com/cvdfoundation/kinetics-dataset**
And I try to modify the parameters in this directory:config/video_classification/kinetics/mobilevit_st.yaml
When I modify the root of the training set and validation set,it seems doesn't load the data correctly!And I use one 1080ti.
![image](https://user-images.githubusercontent.com/87011223/177525340-5f4d749f-1179-4180-8687-b8b97a1cf0f4.png)

Does it need to do other things to load the dataset?
Thank you!!

**Comment by sacmehta** [comment]
Could you please share the dataset organization in your setup?

This codebase assumes structure similar to ImageNet, where each sub-folder inside the training or validation folder is a class. See [here](https://github.com/apple/ml-cvnets/blob/84d992f413e52c0468f86d23196efd9dad885e6f/data/datasets/video_classification/kinetics.py#L66).

**Comment by andrewwang0612** [comment]
OK,I follow the instruction on this github to prepare the dataset
**https://github.com/cvdfoundation/kinetics-dataset**
![image](https://user-images.githubusercontent.com/87011223/177583039-f98bd6a2-ef45-4e37-8e5c-7ee395ed369b.png)
![image](https://user-images.githubusercontent.com/87011223/177583490-9309a4b3-83fd-4a63-b732-89f39fce0e1c.png)
Thank you so much!

**Comment by sacmehta** [comment]
Processing script looks good to me.

could you check if 400 folder corresponding to 400 classes in the Kinetics dataset exists inside the training and validation locations that you are using?

**Comment by andrewwang0612** [comment]
Ok,It seems like I haven't check the dataset problem.
I will check it,Thank you!!

**Comment by sacmehta** [comment]
I assume the issue is solved, so closing it. Feel free to re-open it if issue is still not resolved.

**Comment by andrewwang0612** [comment]
Thank you,the problem is solved.

Sorry,I want to ask another question about the custom dataset and 
the parameters in this directory:config/video_classification/kinetics/mobilevit_st.yaml
![image](https://user-images.githubusercontent.com/87011223/179728936-076b9b51-19f7-427b-93df-e0b020150511.png)
In this file,clips_per_video and num_frames_per_clip are 8,
Assume for 2 videos with 8 and 64 frames,
**The dataset size will be (1+8)?**

If the length of the video larger than 64 frames,
Do I have to seperate the video to shorter segment or I can just modeify the parameter of **clips_per_video**?
